src/rendering/mask/alpha/AlphaMaskPipe.ts

```diff
@@ -30,7 +31,9 @@ class AlphaMaskEffect extends FilterEffect implements PoolItem
         super();
 
         this.filters = [new MaskFilter({
-            sprite: new Sprite(Texture.EMPTY)
+            sprite: new Sprite(Texture.EMPTY),
+            resolution: null,
```

Comment:
This would be nicer, more readable here:
```suggestion
            resolution: 'inherit',
```